primitives/blockcache/iomanager.cpp

```diff
@@ -1359,15 +1359,13 @@ ioManager::ioManager(FileBufferMgr& fbm,
     go();
 }
 
-void ioManager::buildOidFileName(const BRM::OID_t oid, const uint16_t dbRoot, const uint16_t partNum, const uint32_t segNum, char* file_name)
+void ioManager::buildOidFileName(const BRM::OID_t oid, uint16_t dbRoot, const uint16_t partNum, const uint32_t segNum, char* file_name)
```

Comment:
partNum is uint32_t and segNum is uint16_t space in the dbrm code. It is reversed here, please change.

Reply:
Good catch!  This must have been here since the beginning.